nncf/common/graph/layer_attributes.py

```diff
@@ -30,6 +37,9 @@ class BaseLayerAttributes(ABC):
     def __eq__(self, __o: object) -> bool:
         return isinstance(__o, self.__class__) and self.__dict__ == __o.__dict__
 
+    def get_backend_agnostic_attributes(self) -> "BaseLayerAttributes":
+        return self
```

Comment:
For future: move all backend-specific layer attributes e.g. `activation_attributes`, `weights_attributes` to common attributes to minimize differences between backends